### ISSUE TITLE
Requests for pennsieve-agent v1.1.10

### ISSUE THREAD

**Issue by dongbohu** [Pennsieve/pennsieve-agent#3]
### 1. Show current version somewhere?
It would be nice to support a `pennsieve version` command, which would print out the current version (or show the version number at the very beginning of  `-h` output)

### 2. `-h` option has the following output:
```
...
Flags:
      --db string   db file (default is $HOME/.pennsieve/config.ini)
  -h, --help        help for pennsieve
```
Note the `--db string` line is not aligned with the `-h --help` line.

### 3. `pennsieve manifest sync` has the following output: 
```
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/pennsieve/pennsieve-agent/cmd/manifest.glob..func9(0x1584a20?, {0x16175a0?, 0x0?, 0x0?})
	/home/runner/work/pennsieve-agent/pennsieve-agent/cmd/manifest/sync.go:21 +0x3ac
github.com/spf13/cobra.(*Command).execute(0x1584a20, {0x16175a0, 0x0, 0x0})
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.4.0/command.go:860 +0x663
github.com/spf13/cobra.(*Command).ExecuteC(0x1581d20)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.4.0/command.go:974 +0x3b4
github.com/spf13/cobra.(*Command).Execute(...)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.4.0/command.go:902
github.com/pennsieve/pennsieve-agent/cmd.Execute()
	/home/runner/work/pennsieve-agent/pennsieve-agent/cmd/root.go:73 +0x25
main.main()
	/home/runner/work/pennsieve-agent/pennsieve-agent/main.go:21 +0x17
```
It would be more helpful to show a regular help message instead of exposing these implementation details.

### 4. Change log?
Can you maintain a `changelog` file that shows the changes in each new version?

**Comment by dongbohu** [comment]
I got the following output when installing on Ubuntu 20.04 by `dpkg -i` command:
```
Selecting previously unselected package pennsieve-agent.
(Reading database ... 432913 files and directories currently installed.)
Preparing to unpack pennsieve-agent_1.1.10_amd64.deb ...
Unpacking pennsieve-agent (1.1.10) ...
Setting up pennsieve-agent (1.1.10) ...
Install log: /root/.pennsieve/install.log
chown: cannot access 'sora': No such file or directory
chown: cannot access 'dongbo:sora': No such file or directory
chown: cannot access 'sora': No such file or directory
chown: cannot access 'dongbo': No such file or directory
```

The last 4 `chown` lines confused me. Both `dongbo` and `sora` are account names. Why the installation process tried to access these two users?

**Comment by dongbohu** [comment]
A few more comments:
* `~/.pennsieve/agent.log` shouldn't have execution bit enabled.
  ```-rwxr--r-- 1 dongbo dongbo 49 Sep 20 10:25 agent.log```
* Can you rename `~/.pennsieve/agent.lock` to something else (such as `pid.lock`) to make filename completion easier?
* When `pennsieve agent stop` is executed, shouldn't `agent.lock` be removed?

**Comment by jwagenaar** [comment]
@dongbohu this is great! Thank you and yes, we will have a look at these issues and feature requests

**Comment by dongbohu** [comment]
Another request (hope you don't mind): Currently the output of `pennsieve manifest create <local_path>` command is:

```

Manifest ID: 18 Message: Successfully indexed 7 files.
```
When I first saw it, I thought I got 18 messages. So I think it will look better if it's changed to:
```
Manifest ID: 18 
Message: Successfully indexed 7 files.
```
(Note that the leading blank line is also removed because it seems unnecessary)

I also notice that even when `pennsieve manifest create <local_path>` command fails, its exit code is still `0`. Can you change it to a non-zero value?

**Comment by jwagenaar** [comment]
Closing this thread as all issues have been addressed in v1.1.21. (#2 is the default behavior in go cli)

**Comment by dongbohu** [comment]
How about this one?
https://github.com/Pennsieve/pennsieve-agent/issues/3#issuecomment-1255131154
